src/Http/Response/Factory.php

```diff
@@ -7,8 +7,8 @@
 use Illuminate\Support\Str;
 use Dingo\Api\Http\Response;
 use Illuminate\Support\Collection;
-use Illuminate\Pagination\Paginator;
 use Dingo\Api\Transformer\Factory as TransformerFactory;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
```

Comment:
Should be just `Paginator`.

I know Fractal needs a `LengthAwarePaginator` instance but that's assuming everyone will use Fractal. Keep in mind I'm not up to speed on all the pagination changes but this type hint seems to be limiting it to a single type.